### ISSUE TITLE
[BUG] 在0.78版本中运行失败

### ISSUE THREAD

**Issue by yqchilde** [dillonzq/LoveIt#547]
### Describe the bug 描述你遇到的错误
在hugo 0.78版本中运行失败并报错，查看历史问题 #518  发现该问题在>0.75版本中就存在了，希望作者抽空看一下

![image](https://user-images.githubusercontent.com/36953759/98332657-0a6c6500-203a-11eb-8914-1807ba82ae17.png)

**Comment by nekr0z** [comment]
Duplicate of #518

**Comment by yqchilde** [comment]
> Duplicate of #518

Yes, I am currently using hugo version 0.74 for the time being, which is the current solution.

**Comment by mwolfaardt** [comment]
I hope this gets fixed soon
